hybrid/clusterlink/ccloud_source_cluster/zk-kafka-destination.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: platform.confluent.io/v1beta1
+kind: Zookeeper
+metadata:
+  name: zookeeper
+  namespace: destination
+spec:
+  replicas: 3
+  image:
+    application: confluentinc/cp-zookeeper:7.0.0
+    init: confluentinc/confluent-init-container:2.2.1
+  dataVolumeCapacity: 10Gi
+  logVolumeCapacity: 10Gi
+---
+apiVersion: platform.confluent.io/v1beta1
+kind: Kafka
+metadata:
+  name: kafka
+  namespace: destination
+spec:
+  replicas: 3
+  image:
+    application: confluentinc/cp-server:7.0.0
+    init: confluentinc/confluent-init-container:2.2.1
+  dataVolumeCapacity: 10Gi
+  mountedSecrets:
+    - secretRef: password-encoder-secret
+  configOverrides:
+    server:
+      - confluent.cluster.link.enable=true
+      - password.encoder.secret=${file:/mnt/secrets/password-encoder-secret/password_encoder_secret:secret}
```

Comment:
We support this as first class API in 2.3. Can you update this examples?  Please refer here - https://github.com/confluentinc/confluent-kubernetes-examples/blob/master/hybrid/ccloud-topic-clusterlink/zk-kafka.yaml